packages/e2e-cypress/src/index.js

```diff
@@ -40,10 +41,18 @@ module.exports = async function (api) {
         );
       });
     } else {
-      // TODO: add webpack code coverage support
-      // See https://www.npmjs.com/package/istanbul-instrumenter-loader
-      // https://github.com/vuejs/vue-cli/issues/1363#issuecomment-405352542
-      // https://github.com/akoidan/vue-webpack-typescript
+      if (api.hasWebpack === true) {
+        api.extendWebpack((cfg) => {
+          cfg.module.rules.push({
+            test: /\.(js|ts|vue)$/,
+            loader: '@jsdevtools/coverage-istanbul-loader',
+            options: { esModules: true },
+            enforce: 'post',
+            include: path.join(__dirname, '..', '..', '..', '..', 'src'),
```

Comment:
This seems off, the include (provided the exclude isn't enough on its own) should be relative the the user folder, not to this AE location

Reply:
I removed `include` property and I extended `exlcude` with **.quasar** folder. Now it seems to be working with this setting.